vpr/src/base/vpr_context.h

```diff
@@ -149,6 +149,25 @@ struct DeviceContext : public Context {
      */
     t_rr_graph_storage rr_nodes; // autogenerated in build_rr_graph
 
+    /**
```

Comment:
These comments are related to metadata. They should be moved to ``RRGraphBuilder`` and ``RRGraphView``
These dead codes should be removed once this PR is ready to merge.